geth-dev-node/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ethereum/client-go:latest
+COPY genesis.json /app/genesis.json
+RUN echo "p" > /app/password.txt
```

Comment:
Recommend using the empty password, in my experience this simplifies a number of things.  `echo "" > /app/password.txt`

Reply:
I think I did try this, but it seemed to be causing issues at the time.  I'll double-check, though.